test/components/sequence/sync/test_sync_workflow.py

```diff
@@ -414,23 +414,23 @@ def test_finalise4_files_on_disk_not_in_int_manifest_expect_added_to_delete_targ
         # Assert
         assert sync_state[CURRENT_STATE_KEY] == SName.DONE_FINALISING
 
-        # Sample60 is the extra
         path = os.path.join(sync_state[OUTPUT_DIR_KEY], sync_state[OBSOLETE_OBJECTS_FILE_KEY])
         df = pd.read_csv(path)
         r = df.loc[
             (df[FILE_NAME_KEY] == "Sample60_20230614T00453848_a34d8705_R1.fastq") |
             (df[FILE_NAME_KEY] == "Sample70_20230614T00453848_a34d8705_R1.fastq.gz") |
             (df[FILE_NAME_KEY] == "Sample80_20230614T00453848_a34d8705_R1.fasta") |
-            (df[FILE_NAME_KEY] == "Sample90_20230614T00453848_a34d8705_R1.fasta.gz")
-        ]
-        assert len(r.index) == 4
+            (df[FILE_NAME_KEY] == "Sample90_20230614T00453848_a34d8705_R1.fasta.gz") |
+            (df[FILE_NAME_KEY] == "Sample100_20230614T00453848_a34d8705_R1.fq.gz")
```

Comment:
Sample100 needs to be committed. The test is failing.

Reply:
Oops, sorry! Added.